### PR TITLE
telegram-cli: keep arguments passed to the wrapper

### PR DESCRIPTION
**Pull request by rnhmjoj**

### CODE REVIEW

File: pkgs/applications/networking/instant-messengers/telegram-cli/default.nix [hunk 1]
```diff
@@ -11,14 +11,17 @@ stdenv.mkDerivation rec {
...
+    lua5_2 python pkgconfig jansson
+  ];
   installPhase = ''
     mkdir -p $out/bin
     cp ./bin/telegram-cli $out/bin/telegram-wo-key
     cp ./tg-server.pub $out/
     cat > $out/bin/telegram-cli <<EOF
     #!${bash}/bin/sh
-    $out/bin/telegram-wo-key -k $out/tg-server.pub
+    $out/bin/telegram-wo-key -k $out/tg-server.pub "\$@"
```
Comment (lethalman) [inline pkgs/applications/networking/instant-messengers/telegram-cli/default.nix]: That will pass all the arguments as a single argument, right?

Comment (rnhmjoj) [inline pkgs/applications/networking/instant-messengers/telegram-cli/default.nix]: @lethalman No, it won't. I am not a shell expert but `"$@"` should expand to the arguments the script is called with. The number of arguments should unchanged.
I tried with this:
```shell
countargs(){ echo "$#" }
test(){ echo "$#"; countargs "$@" }
```

    $ test a b c
    3
    3

**Comment by lethalman** [comment]
Ok thanks.

File: pkgs/applications/networking/instant-messengers/telegram-cli/default.nix (hunk 1 above)
Comment (rnhmjoj) [inline pkgs/applications/networking/instant-messengers/telegram-cli/default.nix]: I use telegram-cli for this [program](https://gist.github.com/rnhmjoj/5f42ce3f31f8be88e921#file-telegram-hs). It stopped working when I moved to nixOS for this reason but it runs fine with this patch now.

**Pull request merged**